Makefile

```diff
@@ -7,10 +7,10 @@ DOCNAME = UCDlist
 DOCVERSION = 1.5
 
 # Publication date, ISO format; update manually for "releases"
-DOCDATE = 2022-10-05
+DOCDATE = 2023-02-03
```

Comment:
I suppose you mean 2023-03-02 not 2023-02-03.

Reply:
I wonder if the date should be 2023-01-25, the TCG approval for the list PEN -> EN

Reply:
usually, this is the selected date, indeed